run.ps1

```diff
@@ -232,10 +232,14 @@ function Format-LanguageCode {
             $returnCode = 'sk'
             break
         }
-        '^sr' {
+        '^sr$' {
```

Comment:
I think you can specify `^(sr|sr-Cyrl)` here

Reply:
Updated, thanks for the suggestion.